cypress/integration/addNewClient/client.js

```diff
@@ -15,83 +15,89 @@ describe("Test for adding new client ", () => {
   });
 
   it("should verify validation error message while submitting without mandatory fields", () => {
-    const selector = [".js-input-error", ".js-input-error", ".js-input-error"];
-    const errorMessage = [
+    const emptyValidationError = [
       "The First Name field is required.",
       "The Last Name field is required.",
       "The Assignee field is required.",
     ];
 
-    client
-      .clickClient()
-      .wait(1000)
-      .clickAdd()
-      .wait(1000)
-      .clickSave("Save")
-      .dataVerify(selector, "include.text", errorMessage)
-      .clickCancel("Cancel");
+    client.clickOnClientTab();
+
+    cy.wait(1000);
+
+    client.clickOnAddButton();
+
+    cy.wait(3000);
+
+    client.clickOnSaveButton();
+
+    cy.emptyValidation(emptyValidationError);
+
+    client.clickOnCancelButton();
   });
 
   context("form fill for create new client dependent test", () => {
-    const firstname = faker.name.firstName();
-    const lastname = faker.name.lastName();
-    const email = faker.internet.email();
-
     beforeEach(() => {
+      const firstName = faker.name.firstName();
+      const lastName = faker.name.lastName();
+      const email = faker.internet.email();
+      cy.wrap(email).as("email");
+      const name = `${firstName} ${lastName}`;
+      cy.wrap(name).as("name");
+
+      client.clickOnClientTab();
+
+      cy.wait(2000);
+
       client
-        .clickClient()
-        .wait(1000)
-        .clickAdd()
-        .wait(1000)
-        .typeFirstName("1", firstname)
-        .typeLastName("1", lastname)
-        .typeEmail("1", email);
+        .clickOnAddButton()
+        cy.wait(2000)
+
+        client.typeFirstName(firstName)
+        .typeLastName(lastName)
+        .typeEmail(email);
     });
 
-    it("should verify the functionality of cancel button", () => {
-      client.clickCancel("Cancel");
+    it("should verify the functionality of cancel button and data are not added in list", function () {
+      client
+        .selectAssignee()
+        .selectProduct()
+       .clickOnCancelButton()
+        .verificationNameAndEmail("not.have.text", this.email, this.name);
     });
 
-    it("should add new client and verify added data is exist in list", () => {
-      client.clickSave("Save").wait(3000);
+    it("should add new client and verify added data in list", function () {
```

Comment:
you can rename test description as "should add new client and verify added data existence in list"